### PR TITLE
Add TagPullCommandTest

### PR DESCRIPTION
**Pull request by chrisjwelly**
These tests were guided by:
 1. Tagpull a tag that increases all 3 counts
 2. Tagpull a tag that increases 1 of the counts
 3. Tagpull that has no effect
 4. Tagpull invalid index
 5. Tagpull twice the same thing
 6. Tagpull pulls something that already exists - check for distinct
 7. Tagpull multiple tags

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/TagPullCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+                .withSkill(TypicalSkill.GIT)
+                .build();
+
+        CommandResult commandResult = new TagPullCommand(index, tags).execute(model);
+        assertEquals(String.format(feedbackToUser, 2, 2, 2), commandResult.getFeedbackToUser());
+
+        Resume resume = model.getResumeByIndex(index);
+        assertEquals(resume, expectedResume);
+
+        CommandResult commandResultAgain = new TagPullCommand(index, tags).execute(model);
```
Comment (helloImHai) [inline src/test/java/seedu/address/logic/commands/TagPullCommandTest.java:187]: Good check yo

**Review by helloImHai — APPROVED** [review]
As it should be m8 💯💯💯
All tests are made by assumption that you've added to the right resume and all numbers add up right?

**Comment by chrisjwelly** [comment]
> 
> 
> As it should be m8 💯💯💯
> All tests are made by assumption that you've added to the right resume and all numbers add up right?

Yes, I carefully manipulated the TypicalResumeBook and the items added to have some Tags. This will allow the numbers to add up correctly as long as the TypicalResumeBook is not changed and the Tags of the items added aren't modified.

File: src/test/java/seedu/address/testutil/TypicalResumeBook.java [hunk 2]
```diff
@@ -42,6 +42,9 @@
             .withSkill(TypicalSkill.REACT)
             .withResume(TypicalResume.ME_RESUME)
             .build();
+
+    // This TypicalResumeBook is used mainly for the ResumeEdit and TagPull Command tests
```
Comment (helloImHai) [inline src/test/java/seedu/address/testutil/TypicalResumeBook.java:46]: Great work!

**Pull request merged**